### PR TITLE
[FEK-14] Add button go back to top button

### PR DESCRIPTION
**Pull request by skrzepij**
^FEK-14 review

**Preview:**

image

### CODE REVIEW

**Review by skrzepij — COMMENTED** [review]
🚀 🚀 🚀 🚀 🚀 🚀 🚀 
giphy

File: components/tableOfContent/tableOfContent.tsx [hunk 1]
```diff
@@ -7,15 +11,17 @@ type Props = {
 }
 
 export function TableOfContent({ toc }: Props): JSX.Element {
+  const { setVisibility } = React.useContext(TocVisibilityContext)
+
   return (
-    <div className={styles.tableOfContent}>
+    <InView as="div" className={styles.tableOfContent} onChange={(inView) => setVisibility(inView)}>
```
Comment (MateuszWasik) [inline components/tableOfContent/tableOfContent.tsx]: It's a good practice to reduce logic inside render/return as much as possible.  Can we create a different function inside `TableOfContent` and move that logic there ?

**Review by MateuszWasik — APPROVED** [review]

File: components/tableOfContent/tableOfContent.tsx (hunk 1 above)
Comment (skrzepij) [reply components/tableOfContent/tableOfContent.tsx]: On point! 💥 Thanks for that 🙇🏻 

Fixed 👍🏻

Comment (skrzepij) [reply components/tableOfContent/tableOfContent.tsx]: On point! 💥 Thanks for that! 🙇🏻

**Pull request merged**